### ISSUE TITLE
improve consistency of forward and inverse modeling documentation

### ISSUE THREAD

**Issue by robertoostenveld** [fieldtrip/website#571]
As discussed with @mcpiastra. 

The different pages should better use the same terminology, the same (or similar) figures to explain concepts, and link to each other through tags and explicit links in the "see also" sections.

**Comment by mcpiastra** [comment]
Hi, 

some points. I started working on "Tutorial" pages only. 
1. 
As discussed together, the naming of the sections here
![Screenshot from 2022-07-13 11-09-54](https://user-images.githubusercontent.com/13639861/178719454-06644d8d-c51c-49b3-b460-5276abb6a25f.png)
should be more consistent, i.e., 

- Construct a singleshell heamodel for MEG source analysis
- Construct a BEM heamodel for EEG source analysis
- Construct a FEM heamodel for EEG source analysis
- Construct a sourcemodel for MEG or EEG source analysis
- Localizing electrodes using a 3D-scanner
- Reconstruct oscillatory sources in MEG data using a beamformer
- Reconstruct oscillatory sources in combined MEG/EEG data using a beamformer
- Reconstruct visual gamma and cortico-muscular coherence
- Reconstruct event-related fields using minimum-norm estimation
- Reconstruct event-related activity in combined MEG/EEG data with dipole fitting
- Reconstruct epileptic spikes (activity?) in MEG data using a kurtosis beamformer
- Computation of virtual MEG channels in source-space

If I understood correctly, changing the names of the sections above should be done with care because the same name is repeated in several places and there should be consistency.

**Comment by mcpiastra** [comment]
2. I will focus now on making the content of each of those subchapters mentioned above more consistent and similar to each other (e.g., same list of content, same figures)

**Comment by mcpiastra** [comment]
3. tags should be checked. Here:

https://github.com/fieldtrip/website/tree/master/_data/tag 

I can see that there is "source", "sourceanalysis" and "sourcemodel"; "inverse"; "forward"; "headmodel". Are all needed? E.g., "forward" is used only once, "source" and "sourceanalysis" sound the same.

**Comment by robertoostenveld** [comment]
agreed that https://www.fieldtriptoolbox.org/tag/sourceanalysis/ should be merged with https://www.fieldtriptoolbox.org/tag/source/
